src/wallet/rpcwallet.cpp

```diff
@@ -2923,6 +2928,236 @@ UniValue bumpfee(const JSONRPCRequest& request)
     return result;
 }
 
+UniValue getlabeladdress(const JSONRPCRequest& request)
+{
+    CWallet * const pwallet = GetWalletForJSONRPCRequest(request);
+    if (!EnsureWalletIsAvailable(pwallet, request.fHelp)) {
+        return NullUniValue;
+    }
+
+    if (request.fHelp || request.params.size() != 1)
+        throw std::runtime_error(
+            "getlabeladdress \"label\"\n"
+            "\nReturns the current 'label address' for this label.\n"
+            "\nArguments:\n"
+            "1. \"label\"       (string, required) The label for the address. It can also be set to the empty string \"\" to represent the default label.\n"
+            "\nResult:\n"
+            "\"bitcoinaddress\"   (string) The 'label address' for the label\n"
+            "\nExamples:\n"
+            + HelpExampleCli("getlabeladdress", "")
+            + HelpExampleCli("getlabeladdress", "\"\"")
+            + HelpExampleCli("getlabeladdress", "\"mylabel\"")
+            + HelpExampleRpc("getlabeladdress", "\"mylabel\"")
+        );
+
+    LOCK2(cs_main, pwallet->cs_wallet);
+
+    // Parse the label first so we don't generate a key if there's an error
+    std::string strLabel = AccountFromValue(request.params[0]);
```

Comment:
It'd be nice to eventually rename this function `LabelFromValue()`. That can be done in a follow-up PR.